packages/block-editor/src/utils/transform-styles/index.js

```diff
@@ -19,18 +14,10 @@ import wrap from './transforms/wrap';
  */
 const transformStyles = ( styles, wrapperClassName = '' ) => {
 	return Object.values( styles ?? [] ).map( ( { css, baseURL } ) => {
-		const transforms = [];
-		if ( wrapperClassName ) {
-			transforms.push( wrap( wrapperClassName ) );
-		}
-		if ( baseURL ) {
-			transforms.push( urlRewrite( baseURL ) );
-		}
-		if ( transforms.length ) {
-			return traverse( css, compose( transforms ) );
-		}
-
-		return css;
+		return postcss( [
+			wrap( { scopeTo: wrapperClassName } ),
+			rebaseUrl( { rootUrl: baseURL } ),
+		] ).process( css, {} );
```

Comment:
`.process` returns a `LazyResult` (Promise). Reference: https://postcss.org/api/#processor-process

Well, this is unfortunate... We need to make this sync 😢

Reply:
Yes, the sync variant should also work.